pkg/cluster/routing/routing.go

```diff
@@ -65,21 +75,24 @@ func (m Meta) Len() int {
 	return capnp.TextList(m).Len()
 }
 
+func (m Meta) At(i int) (Field, error) {
```

Comment:
When will the `At` function be used externally? Won't we always use `Get`?

Reply:
At lets us view the keys.  I thought about having a `Keys() []string` method, but it somehow seemed more convoluted than just providing the key-value pair directly (since that's how it's encoded).

Reply:
Just to clarify:  one place where we'll want to list all the keys is `ww client ls -l /` (notice the `-l` flag).  Here, we want to print all the key-value metadata pairs for each host, so we'll need a way to iterate through keys.